src/main/java/frc/robot/subsystems/arm/ManualMoveState.java

```diff
@@ -32,7 +32,7 @@ public void execute() {
 
         // TODO: figure out how to manually control arm
         
-        double wristInput = RobotMap.arm.applyDeadband(RobotMap.manipulatorController.getRawAxis(ManipulatorControls.WRIST_BACKWARD_AXIS)) 
+        /*double wristInput = RobotMap.arm.applyDeadband(RobotMap.manipulatorController.getRawAxis(ManipulatorControls.WRIST_BACKWARD_AXIS)) 
```

Comment:
What's the logic behind commenting this out? Also, comment out large blocks of code like this by highlighting the block and pressing `Ctrl + /`.